src/DefaultHandle.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles, withStylesPropTypes } from 'react-with-styles';
+
+import {
+  DEFAULT_HANDLE_WIDTH_UNITS,
+  BACKGROUND_HEIGHT_UNITS,
+  VERTICAL,
+} from './constants/SliderConstants';
+
+import handlePropTypes, { handleDefaultProps } from './propTypes/HandlePropTypes';
+
+export const propTypes = {
+  ...withStylesPropTypes,
+  ...handlePropTypes,
+  'aria-valuetext': PropTypes.string,
+  'aria-label': PropTypes.string,
+};
+
+const defaultProps = {
+  ...handleDefaultProps,
+  'aria-valuetext': undefined,
+  'aria-label': undefined,
+};
+
+function DefaultHandle({
+  css,
+  styles,
+  orientation,
+  disabled,
+  ...passProps
+}) {
+  const {
+    handleRef,
+    theme,
+    ...rest
+  } = passProps;
+  return (
+    <button
+      ref={handleRef}
+      {...css(
+        styles.DefaultHandle_handle,
+        orientation === VERTICAL
+        ? styles.DefaultHandle_handle__vertical
+        : styles.DefaultHandle_handle__horizontal,
+        disabled && styles.DefaultHandle_handle__disabled,
+      )}
+      {...rest}
+    />
+  );
+}
+DefaultHandle.propTypes = propTypes;
+
+DefaultHandle.defaultProps = defaultProps;
+
+export default withStyles(({ color, unit }) => ({
+  DefaultHandle_handle: {
+    width: DEFAULT_HANDLE_WIDTH_UNITS * 3.8 * unit,
+    height: DEFAULT_HANDLE_WIDTH_UNITS * 3.8 * unit,
+    borderWidth: 1,
+    borderStyle: 'solid',
+    borderColor: color.grey,
+    backgroundColor: color.white,
+    borderRadius: '20%',
+    outline: 'none',
+    zIndex: 2,
+    boxShadow: `0 ${unit / 4}px ${unit / 4}px ${color.textDisabled}`,
+    ':focus': {
+      boxShadow: `${color.focus} 0 0 2px 2px`,
```

Comment:
Should this also use units?

Reply:
Will update all CSS related issues in a separate PR! (ie: pulling things out into theme, addressing weird uniting from internal version, etc. )